### DISCUSSION TITLE
Plans for global search?

### DISCUSSION THREAD

**Discussion by RyanGreenburg** [deliton/eldenring-api#5] [Q&A] [answered]
Hey, first I just want to say this is a fantastic API. Thank you for taking the time to put it together. I'm curious if you have plans to add in search functionality?

**Comment by deliton** [comment, marked as answer]
Hey @RyanGreenburg ! We already have search for both REST and GraphQL, but it only searches in one domain. Initially I didn't think of adding a Global search (all types of objects) but now that you commented, I can see a lot of use cases.
